### PR TITLE
WIP: Remove trailing whitespace from all files

### PR DESCRIPTION
**Pull request by shmup**
I've stripped all trailing whitespace from the project. It shaves off 46333 bytes!

### CODE REVIEW

File: libc/unicode/kcombiningchars.S [hunk 1]
```diff
@@ -75,7 +75,7 @@ kCombiningCharsLz4:
 	.byte	0x00,0xa0,0x57,0x00,0x50,0xfe,0x7f,0xdf	# áW P■⌂▀
 	.byte	0xe0,0xff,0x41,0x01,0x28,0x1f,0x40,0x2f	#αλA☺(▼@/
 	.byte	0x00,0xff,0x00,0xe0,0xfd,0x66,0x00,0x00	# λ α²f  
-	.byte	0x00,0xc3,0x01,0x00,0x1e,0x00,0x64,0x20	# ├☺ ▲ d
```
Comment (mnhrdt) [inline libc/unicode/kcombiningchars.S:78]: the trailing whitespaces on this file seem alright

Comment (shmup) [reply libc/unicode/kcombiningchars.S]: I've went ahead and restored this file. Thanks for :eyes:

**Comment by jart** [comment]
Thank you for spotting this and taking the time to send a PR. Could you please look into the CI failure with MbedTLS parsing?

Welcome as a new contributor. We need to go through one quick hurdle before we can proceed. Could you please email Justine Tunney <jtunney@gmail.com> and say that you intend to assign her the copyright to the changes you contribute to Cosmopolitan? Please send the email under your legal name rather than anonymously. See [CONTRIBUTING.md](CONTRIBUTING.md) for further details. We make the ask because it helps us ship the tiniest most legally permissive prim and proper binaries. You also only need to do it one time, so any future changes you send us can be quickly merged.

Thanks!

**Comment by shmup** [comment]
> Thank you for spotting this and taking the time to send a PR. Could you please look into the CI failure with MbedTLS parsing?

I incorrectly trimmed whitespace from a file that was supposed to fail because of said whitespace. All good now!

**Pull request merged**